Gems/EMotionFX/Code/Tests/UI/UIFixture.h

```diff
@@ -51,6 +51,8 @@ namespace EMotionFX
 
     protected:
         QApplication* m_uiApp = nullptr;
+    private:
+        static inline int s_argc{0};
```

Comment:
Why is this variable needed?

Reply:
The EMotionFX UI tests create a `QApplication` instance. The documentation for `QApplication` states:
> Warning: The data referred to by argc and argv must stay valid for the entire lifetime of the QApplication object.

Previously, the value passed to `argc` was stack allocated, and not valid for the duration of the QApplication object.